lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -635,6 +635,18 @@ html.video-fullscreen {
         border-bottom: 1px solid #ddd;
         margin-bottom: ($baseline*0.75);
         padding: 0 0 15px;
+
+        .problem-reset-btn {
+          border: none;
+          max-width: 210px;
+          position: absolute;
```

Comment:
Why is it absolute? This way it's "outside" of the `.vert`, so it's getting clipped in the edx/frontend-app-learning MFE.
We could e.g. enclose this button in a `div` (please give it a proper class name) and set the button's position to `relative`.

![image](https://user-images.githubusercontent.com/3189670/134905636-18852a98-a0da-4eed-afea-c927e9deb111.png)